core-parser/src/main/java/org/uniprot/core/parser/tsv/uniparc/UniParcEntryLightValueMapper.java

```diff
@@ -0,0 +1,118 @@
+package org.uniprot.core.parser.tsv.uniparc;
+
+import org.uniprot.core.parser.tsv.EntityValueMapper;
+import org.uniprot.core.uniparc.UniParcEntryLight;
+import org.uniprot.core.uniprotkb.taxonomy.Organism;
+
+import java.time.LocalDate;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class UniParcEntryLightValueMapper implements EntityValueMapper<UniParcEntryLight> {
```

Comment:
We removed fields in the config, Do you think we need to clean code from UniParcEntryValueMapper to reflect the column option changes?

Reply:
Which fields did we remove? I didn't get what cleanup are you talking about?